scripts/feature_tests/upgrade/Makefile

```diff
@@ -8,5 +8,4 @@ all: upgrade
 
 .PHONY: upgrade
 upgrade:
-	wget -q https://raw.githubusercontent.com/Nordix/airship-dev-tools/master/upgrade_tests/upgrade.sh
 	bash upgrade.sh
```

Comment:
This is duplicated with https://github.com/metal3-io/metal3-dev-env/pull/357/files#diff-2ccc6e188d7155eaf1e4aa37e6138a17L11.

Reply:
Yes, PRs 357 and 390 have "common" duplicated files because they are being executed in parallel before merging.